assignments/prototypes.js

```diff
@@ -15,14 +15,27 @@
   * dimensions (These represent the character's size in the video game)
   * destroy() // prototype method that returns: `${this.name} was removed from the game.`
 */
-
+function GameObject(attribute) {
+  this.createdAt = attribute.createdAt;
+  this.name = attribute.name;
+  this.dimensions = attribute.dimensions;
+  this.destroy = function () {
+    return `${this.name} was removed from the game.`;
+  };
+}
 /*
   === CharacterStats ===
   * healthPoints
   * takeDamage() // prototype method -> returns the string '<object name> took damage.'
   * should inherit destroy() from GameObject's prototype
 */
-
+function CharacterStats(attribute) {
+  this.healthPoints = attribute.healthPoints;
+  this.takeDamage = function () {
+    return `${this.name} took damage.`
+  }
+  
+}
```

Comment:
This also is not set up correctly.  Here is the right way:

function CharacterStats(stats) {
  GameObject.call(this, stats); 
  this.hp = stats.hp;
  this.name = stats.name;
}

CharacterStats.prototype = Object.create(GameObject.prototype); // this inherits the destroy function from the GameObject prototype

CharacterStats.prototype.takeDamage = function() {
  return `${this.name} took damage.`;  
}